js/src/modals/CreateAccount/store.js

```diff
@@ -91,7 +91,11 @@ export default class Store {
   }
 
   @computed get hasAddress () {
-    return !!(this.address);
+    return this.address.length !== 0;
```

Comment:
You are right, will revert - extra safety. (It _should_ be empty, but no need to muck about)